kolibri/core/auth/api.py

```diff
@@ -573,6 +574,7 @@ def _map_dataset(facility):
     return dataset
 
 
+@method_decorator(csrf_protect, name="dispatch")
```

Comment:
Not sure this is needed here, as this is covered by authentication permissions? Did you find otherwise?

Reply:
Yes you are correct. I was thinking if it contains create_facility endpoint then that must be done by a browser right?

Reply:
do you want it removed?

Reply:
Yes, let's remove it - we only need to add it in places where Django REST Framework is not already doing it for us.

Reply:
yes done